SPECS/pango/pango.spec

```diff
@@ -1,58 +1,68 @@
-Summary:	library for laying out and rendering of text.
-Name:		pango
-Version:	1.40.4
-Release:    4%{?dist}
-License:	LGPLv2 or MPLv1.1
-URL:		http://pango.org
-Group:		System Environment/Libraries
+Summary:        library for laying out and rendering of text.
+Name:           pango
+Version:        1.44.7
+Release:        1%{?dist}
+License:        LGPLv2 OR MPLv1.1
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	    https://download.gnome.org/sources/pango/1.40/%{name}-%{version}.tar.xz
-BuildRequires:	glib-devel
-BuildRequires:	cairo
-BuildRequires:	cairo-devel
-BuildRequires:	libpng-devel
-BuildRequires:	fontconfig
-BuildRequires:	fontconfig-devel
-BuildRequires:	harfbuzz
-BuildRequires:	harfbuzz-devel
-BuildRequires:	freetype
-Requires:	    harfbuzz-devel
+Group:          System Environment/Libraries
+URL:            https://pango.org
+Source0:        https://download.gnome.org/sources/pango/1.44/%{name}-%{version}.tar.xz
+BuildRequires:  cairo
+BuildRequires:  cairo-devel
+BuildRequires:  fontconfig
+BuildRequires:  fontconfig-devel
+BuildRequires:  freetype
+BuildRequires:  glib-devel
+BuildRequires:  gobject-introspection-devel
+BuildRequires:  harfbuzz
+BuildRequires:  harfbuzz-devel
+BuildRequires:  libpng-devel
+BuildRequires:  meson
+BuildRequires:  pkg-config
+BuildRequires:  pkgconfig(fribidi)
```

Comment:
Why is this required to build now? I'm curious if it's related to the new `*.typelib` files being produced and maybe we can disable their generation along with a dependency on `pkgconfig(fribidi)` in one go.

Reply:
This is also a change due to the upgrade of the pango version. In the meson.build file of the new pango source code, it lists fribidi as a dependency for pango and the version of fribidi that pango is looking for.